### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by tigerw**
Minor changes:
* Group related styles together in CSS
* Try to make the architecture names consistent to ones without spaces

### CODE REVIEW

**Review by mathiascode — APPROVED** [review]
Thanks, looks fine. I don't have a strong opinion on the naming, but I'll leave that to @bearbin.

File: download-links.html [hunk 1]
```diff
@@ -18,23 +18,23 @@ <h2>Source Downloads</h2>
...
-<li>64 bit x86: <a href="https://download.cuberite.org/windows-x86_64/Cuberite.zip">https://download.cuberite.org/windows-x86_64/Cuberite.zip</a></li>
+<li>x86: <a href="https://download.cuberite.org/windows-i386/Cuberite.zip">https://download.cuberite.org/windows-i386/Cuberite.zip</a></li>
+<li>x86-64: <a href="https://download.cuberite.org/windows-x86_64/Cuberite.zip">https://download.cuberite.org/windows-x86_64/Cuberite.zip</a></li>
 </ul>
 
 <h2>macOS Downloads</h2>
 
 <ul>
-<li>64 bit x86: <a href="https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz">https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz</a></li>
+<li>x86-64: <a href="https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz">https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz</a></li>
```
Comment (bearbin) [inline download-links.html]: What did I say in the last PR. IIAB, DFI! I can't see any benefit to changing these: they are perfectly legible for all our users, and the proposed replacements are not any more consistent (in fact, less so than before!).

File: index.html [hunk 2]
```diff
@@ -20,7 +20,7 @@ <h2>Download for Minecraft Java Edition 1.8 - 1.12.2</h2>
 							<a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">32-bit</a>
 							<a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">64-bit</a>
```
Comment (bearbin) [inline index.html]:
```suggestion
							<a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">x86 32-bit</a>
							<a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">x86 64-bit</a>
```

Now we have an ARM 64 bit, we do actually need to add this, for disambiguation.

**Review by bearbin — CHANGES_REQUESTED** [review]

**Comment by tigerw** [comment]
Broken is in the eye of the beholder, as they say. Since the download-links page is targeted at more technically-minded users, I propose we model the names off what Debian names their downloads:

**amd64** · **arm64** · armel · **armhf** · **i386** · mips64el · mipsel · ppc64el · s390x

That means "32 bit x86" and "64 bit x86" is out of place compared to the others.

For the homepage, I propose we model it off the Firefox download names. Here we'd use "32-bit" and "64-bit" with hyphens, but qualify with an architecture.

**Comment by bearbin** [comment]
That sounds fine (although our i386 is actually i686, that's because we inherit the [misname from debian](https://lists.debian.org/debian-devel-announce/2016/05/msg00001.html)).

It will mean more work though to update the buildserver and download links themselves so they're consistent with the names we give them on the download links page.

**Comment by tigerw** [comment]
Ah I see, then let's forget about download-links for now and I'll just use your suggestion for the home page.

**Review by bearbin — APPROVED** [review]

**Comment by tigerw** [comment]
Thanks all!

**Pull request merged**